plugins/flutter_aepedgebridge/README.md

```diff
@@ -0,0 +1,141 @@
+> 
+> However, in cases where it is not easy to refactor an existing application, the Edge Bridge extension exists as a drop-in solution to send converted `trackState` and `trackAction` events to the Edge Network.
+
+## Prerequisites
+
+The Edge Bridge extension has the following peer dependencies, which must be installed prior to installing the Edge Bridge extension:
+
+- [flutter_aepcore](../flutter_aepcore/README.md)
+- [flutter_aepedge](../flutter_aepedge/README.md)
+- [flutter_aepedgeidentity](../flutter_aepedgeidentity/README.md)
+
+## Installation
+
+Install instructions for this package can be found [here](https://pub.dev/packages/flutter_aepedgebridge/install).
+
+> Note: After you have installed the SDK, don't forget to run `pod install` in your `ios` directory to link the libraries to your Xcode project.
+
+## Tests
+
+Run:
+
+```bash
+flutter test
+```
+
+## Usage
+
+### Installing and registering the extension with the AEP Mobile Core
+
+Install the Adobe Experience Platform Edge Network extension in your mobile property and configure the default Datastream ID by following the steps in the [Edge Network extension documentation](https://developer.adobe.com/client-sdks/documentation/edge-network).
+
+> **Note**  
+Experience Platform Edge Bridge does not have a corresponding extension card in the Data Collection UI; no changes to a Data Collection mobile property are required to use Edge Bridge.
+
+### Registering the extension with AEPCore:
+
+ > **Note**  
+ It is required to initialize the SDK via native code inside your AppDelegate (iOS) and MainApplication class (Android).
+
+As part of the initialization code, make sure that you set the SDK wrapper type to `Flutter` before you start the SDK.
+
+Refer to the [Initialization](https://github.com/adobe/aepsdk_flutter#initializing) section of the root README for more information about initializing the SDK.
+
+**Initialization Example**
+
+iOS
+```objc
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+@import AEPEdgeBridge;
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore setWrapperType:AEPWrapperTypeFlutter];
+
+     // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+    NSString* ENVIRONMENT_FILE_ID = @"YOUR-APP-ID";
+    
+    NSArray *extensionsToRegister = @[AEPMobileEdgeIdentity.class, 
+                                      AEPMobileEdge.class,                                              
+                                      AEPMobileEdgeBridge.class
+                                      ];
+
+    [AEPMobileCore registerExtensions:extensionsToRegister completion:^{
+    [AEPMobileCore configureWithAppId: ENVIRONMENT_FILE_ID];
```

Comment:
nit: formatting - can you add a tab for this line since it is inside the callback